pkg/content/gunzip.go

```diff
@@ -9,10 +9,22 @@ import (
 )
 
 // NewGunzipWriter wrap a writer with a gunzip, so that the stream is gunzipped
-func NewGunzipWriter(writer content.Writer, blocksize int) content.Writer {
+//
+// By default, it calculates the hash when writing. If the option `skipHash` is true,
+// it will skip doing the hash. Skipping the hash is intended to be used only
+// if you are confident about the validity of the data being passed to the writer,
+// and wish to save on the hashing time.
+func NewGunzipWriter(writer content.Writer, blocksize int, opts ...WriterOpt) content.Writer {
```

Comment:
Does it make sense to make blocksize an option too? This code has not yet been released yet

Reply:
Yes! Really good idea! I am putting this in right now. A few minutes...

Reply:
Done.